### PR TITLE
FS 1385 update speed bump page

### PR DESCRIPTION
**Pull request by ramsharma-prog**
Screenshot 2022-09-09 at 11 10 35

### CODE REVIEW

File: frontend/magic_links/templates/landing.html [hunk 1]
```diff
@@ -1,21 +1,70 @@
...
+    <p>This form is split into sections based on how it will be assessed.</p>
+    <p>You do not have to complete it in order, or all at once.</p>
+    <p>
+      You can preview the
+      <a href="https://fsd-funding-prototype-v2.herokuapp.com/cof-w2-all-questions" class="govuk-link">full list of
+        application questions.</a>
+    </p>
+    <p>
+      If you're inactive for 20 minutes, you'll be signed out and lose any
+      information you have not saved.
```
Comment (gidsg) [inline frontend/magic_links/templates/landing.html:30]: I don't think this is true? We're not using he timeout overlay are we @harryyo?

File: frontend/magic_links/templates/landing.html [hunk 2]
```diff
@@ -1,21 +1,70 @@
...
+      "href": url_for('api_MagicLinksView_use', link_id=link_id),
+      "isStartButton": true })
+    }}
+    <h2 class="govuk-heading-m">How to complete your application</h2>
+    <h3 class="govuk-heading-s">Moving through the form</h3>
+    <p>This form is split into sections based on how it will be assessed.</p>
+    <p>You do not have to complete it in order, or all at once.</p>
+    <p>
+      You can preview the
+      <a href="https://fsd-funding-prototype-v2.herokuapp.com/cof-w2-all-questions" class="govuk-link">full list of
```
Comment (gidsg) [inline frontend/magic_links/templates/landing.html:25]: We shouldn't be linking to the prototype, can we just make this link `#` for now?

Comment (ramsharma-prog) [reply frontend/magic_links/templates/landing.html:25]: will do

File: frontend/magic_links/templates/landing.html (hunk 1 above)
Comment (ramsharma-prog) [reply frontend/magic_links/templates/landing.html:30]: It is soon the prototype 🤔 
https://fsd-funding-prototype-v2.herokuapp.com/s20-application/start-page-english

Comment (gidsg) [reply frontend/magic_links/templates/landing.html:30]: @ramsharma-prog I agree the content is correct as per the prototype, but needs revising as this is not how the application actually behaves I believe CC @harryyo

Comment (ramsharma-prog) [reply frontend/magic_links/templates/landing.html:30]: Okay 👍

**Review by gidsg — APPROVED** [review]
👍

**Review by harryyo — APPROVED** [review]
Looks great!

**Pull request merged**